google/cloud/bigquery/job.py

```diff
@@ -34,7 +34,6 @@
 from google.cloud.bigquery.encryption_configuration import EncryptionConfiguration
 from google.cloud.bigquery.external_config import ExternalConfig
 from google.cloud.bigquery.external_config import HivePartitioningOptions
-from google.cloud.bigquery.opentelemetry_tracing import create_span
```

Comment:
Can we remove this import in client.py too? (GitHub doesn't let me comment too far from the diff)

Reply:
Can't remove from client.py because it is used in the _call_api method in the module